rules/integrations/aws/privilege_escalation_iam_update_assume_role_policy.toml

```diff
@@ -125,7 +126,7 @@ reference = "https://attack.mitre.org/tactics/TA0004/"
 
 [rule.new_terms]
 field = "new_terms_fields"
-value = ["cloud.account.id", "user.name", "aws.cloudtrail.flattened.request_parameters.roleName"]
+value = ["cloud.account.id", "user.name", "target.entity.id"]
```

Comment:
`target.entity.id` is a MV field? Do we know if values are susceptible to change, causing unexpected events?